contracts/interfaces/IBasketHandler.sol

```diff
@@ -75,6 +88,13 @@ interface IBasketHandler is IComponent {
     /// @return status The worst CollateralStatus of all collateral in the basket
     function status() external view returns (CollateralStatus status);
 
+    /// Track the basket status changes
+    /// @custom:protected
```

Comment:
This is not actually protected anymore, correct?

Reply:
Missed this one. Will fix.